test/integration/integration_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 The Federation v2 Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/kubernetes-sigs/federation-v2/test/integration/framework"
+)
+
+var FedFixture *framework.FederationFixture
+
+// TestIntegration provides a common setup and teardown for all the integration tests.
```

Comment:
(No action required) If the goal is shared setup, I'm not sure why manually composing a test suite would be the strategy.  Seems like administrivia when we already have automated test discovery.   I don't care too much, though, since I expect integration to be deprecated before too long in preference to managed e2e.

Reply:
This  provides the mechanism to let the subtests run in parallel.

Reply:
Why does shared fixture instantiation/teardown require the usage of a suite?  AFAICT a test calling  `t.Parallel` doesn't require that the test be called as part of a suite.